src/io/filesystem.sample-content.ts

```diff
@@ -0,0 +1,15 @@
+export { INITIAL_CONTENT } from '../features/textEditor/components/tipTapEditorConfigs';
```

Comment:
I think you move `INITIAL_CONTENT` here, there is no need to keep it in `tipTapEditorConfigs`.